cmd/fyneterm/main.go

```diff
@@ -22,6 +22,11 @@ import (
 
 const termOverlay = fyne.ThemeColorName("termOver")
 
+var (
```

Comment:
These look like they should be using constants and not vars?

Reply:
I did think about that, but someone may want to change these,default or such during a build.

Reply:
Is global variables the right way to expose this though? It's not a pattern generally embraced.

Reply:
Yeah, we should probably avoid global variables unless they are constants